packages/scripts/package.json

```diff
@@ -50,7 +50,7 @@
     "file-loader": "^2.0.0",
     "html-webpack-plugin": "3.2.0",
     "jest": "^24.0.0",
-    "jest-environment-jsdom": "^24.0.0",
+    "jest-environment-jsdom": "^23.4.0",
```

Comment:
Might want to align with @jsomsanith here: https://github.com/Talend/ui/pull/2059
And update one or the other PR

Reply:
Good catch, no pb, we'll have a conflict :D